citi-onboarding/DADe-onepage
# feat(feature/create-organization-backend-connection): Created organiz…

## What I did

- Added backend connection to the Organization view
- Adjusted text-aling on Organization Text

## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout develop`
   2. `git pull`
   3. `git checkout feature/create-organization-backend-connection`
   4. `git pull origin feature/create-organization-backend-connection`
   5. `npm install`
   6. `cd client/`
   7. `npm start`

2. Open another terminal and run Keystone server

3. In your browser go to http://localhost:3000 (should open automatically when running `npm start`)

4. Check if the component (Contact) looks just like the mockup in different screen sizes (minimum width: 1366px) and if when you increase it past 1920px it remains the same size (Doesnt grow along with the screen).

5. In your browser, go to http://localhost:3001/admin/organizacao-internas

6. Change the images and text of the list items, and see if it displays correctly after your changes and if the changes actually happen

7. After your changes, make sure to return the list items to its original state (text and image).

8. Read the changed files section on GitHub 

9. Test the things that you think are worth testing, even the ones that are not in this description
